### ISSUE TITLE
Hardware not receiving or transmitting. 

### ISSUE THREAD

**Issue by jamine16** [luispl77/433Screen-SignalHacker#2]
I have build my device according to your diagrams, but it is not receiving or transmitting any data.
I have triple checked all my connections and they are not the issue.

I selected DOIT ESP32 DEVKIT V1 board in Arduino IDE.

Any help would be appreciated.

Thanks

**Comment by jamine16** [comment]
After replacing both RFM modules and all the wiring i found the problem.

In the diagrams it shows the SCK pins of the  SD Card and RFM only connected to each other. After connecting them to pin18 (CLK) on the ESP32, everything is working fine.

**Comment by luispl77** [comment]
> I have build my device according to your diagrams, but it is not receiving or transmitting any data. I have triple checked all my connections and they are not the issue.
> 
> I selected DOIT ESP32 DEVKIT V1 board in Arduino IDE.
> 
> Any help would be appreciated.
> 
> Thanks

Yes thank you for pointing that out, sorry for the late response. The missing "SCK" NET was fixed in the next PCB version.
